README.md

```diff
@@ -11,81 +13,36 @@ Client library for accessing Gro Intelligence's agricultural data platform.
 
 ## Install Gro API client packages
 
-There are two installation methods: using pip, or using git. 
-
-### Install with pip install (preferred)
-
 ```sh
-pip install git+https://github.com/gro-intelligence/api-client.git [--target=<your-gro-path>]
-```
-
-If you don't specify your gro installation path, it will be the local [site-packages]( https://stackoverflow.com/questions/31384639/what-is-pythons-site-packages-directory) directory.
-
-### Install with git clone 
-
-```
-git clone https://github.com/gro-intelligence/api-client.git  [<your-gro-path>]
+pip install git+https://github.com/gro-intelligence/api-client.git
```

Comment:
you could add a link to installing pip since we aren't giving the alternate option

Reply:
pip is in the `Prerequisites` section already, which I did not change

And besides, `pip` is included in Python by default in Python >=2.7.9 & Python 3 >=3.4 (see https://pip.pypa.io/en/stable/installing/). We enforce a minimum version of 2.7.6 in https://github.com/gro-intelligence/api-client/blob/development/setup.py#L18 though we haven't targeted a minimum 3.x version yet. So, it is possible they might not have pip, but it should be a small cross-section